test/unit/apple-pay.test.js

```diff
@@ -150,6 +151,29 @@ function applePayTest (integrationType, requestMethod) {
         });
       });
 
+      describe('when Apple Pay version not supported', function () {
+        let originalMethod = ApplePaySessionStub.supportsVersion;
+        beforeEach(function () {
+          ApplePaySessionStub.supportsVersion = () => false;
```

Comment:
Here we can leverage the existing Sinon sandbox. Can remove the afterEach since the sandbox is torn down afterward.

```
this.sandbox.stub(ApplePaySessionStub, 'supportsVersion').returns(false);
```